### DISCUSSION TITLE
Student tier copilot not showing up, but I am registered as a student and approved?

### DISCUSSION THREAD

**Discussion by shep-analytics** [community/community#46073] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi,

I am a student and registered with GitHub (you can tell b/c free pro); however, it seems that it will still charge me after my copilot trial expires?

<img width="820" alt="image" src="https://user-images.githubusercontent.com/63129428/216205057-8252fa16-d667-4ac7-aeae-34faf865fb26.png">

**Comment by davecheney** [comment, marked as answer]
@shep-analytics you will need to cancel your trial subscription, then go to https://github.com/github-copilot/free_signup to claim you student subscription.
